src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1897,7 +1931,6 @@ def load_data(self):
                     ext_data = pickle.load(ext_data_file.open("rb"))
                 else:
                     continue
-                self.data[ext_data_name] = ext_data
```

Comment:
this is wrong!!! Many extensions have multiple files they need to load (and that's why tests are fortunately failing!) :)

I'll push a fix with some other changes if you don't mind @jonahpearl

Reply:
ah, good catch!